decryptor/mysql/response_proxy.go

```diff
@@ -773,6 +773,12 @@ func (handler *Handler) PreparedStatementResponseHandler(ctx context.Context, pa
 	}
 
 	handler.resetQueryHandler()
+	// if prams_num > 0 params definition block will follow
+	// https://dev.mysql.com/doc/internals/en/com-stmt-prepare-response.html
+	if response.ParamsNum > 0 {
```

Comment:
maybe we should validate that we have enough bind/select items? 
or at least pass paramsCount and validate too as columnsNum?